Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_fun.yml

```
@@ -71,10 +71,10 @@
   - type: SolutionContainerManager
     solutions:
       beaker:
-        maxVol: 1000
+        maxVol: 1200 # Carpmosia-edit - extendedReagentStorage
```

This is set on the parent.
```suggestion
        #        maxVol: 1200 # Carpmosia-edit - extendedReagentStorage
```